client/src/index.tsx

```diff
@@ -58,7 +58,16 @@ if (window.location.hash) {
   ReactDOM.createRoot(root).render(
     <WagmiProvider config={wagmiConfig}>
       <QueryClientProvider client={clientReactQuery}>
-        <RainbowKitProvider modalSize="compact">
+        <RainbowKitProvider
```

Comment:
Add `id` prop and use it to set custom styles.

Reply:
Done.